collections/_article/maintenance-release-godot-3-6-1.md

```diff
@@ -0,0 +1,160 @@
+- Revert changes to `Range::set_value` #65101 ([GH-100459](https://github.com/godotengine/godot/pull/100459)).
+- [3.x, RTL] Track external changes in the custom fonts set by BBCode / `push_font` ([GH-105266](https://github.com/godotengine/godot/pull/105266)).
+- Revert "Fix Button not listing `hover_pressed` stylebox" ([GH-107696](https://github.com/godotengine/godot/pull/107696)).
+
+#### Input
+
+- Fix Xbox Controller on Android ([GH-106021](https://github.com/godotengine/godot/pull/106021)).
+
+#### Physics
+
+- Fix physics platform behaviour regression ([GH-97316](https://github.com/godotengine/godot/pull/97316)).
+
+#### Porting
+
+- Disable Nahimic code injection ([GH-99388](https://github.com/godotengine/godot/pull/99388)).
+
+#### Rendering
+
+- Ameliorate performance regression due to directional shadow `fade_start` ([GH-99536](https://github.com/godotengine/godot/pull/99536)).
+- Hide last DirectionalLight shadow split distance property when using PSSM 3 Splits ([GH-99554](https://github.com/godotengine/godot/pull/99554)).
+- Physics Interpolation - Add editor warning for non-interpolated `PhysicsBody` ([GH-103355](https://github.com/godotengine/godot/pull/103355)).
+
+#### Thirdparty
+
+- certs: Sync with Mozilla bundle as of Oct 19, 2024 ([GH-98855](https://github.com/godotengine/godot/pull/98855)).
+- Update the `Thirdparty` section of `CHANGELOG.md` ([GH-99692](https://github.com/godotengine/godot/pull/99692)).
+- mbedtls: Update to upstream version 2.28.9 ([GH-100013](https://github.com/godotengine/godot/pull/100013)).
+- mbedTLS: Update to version 2.28.10 ([GH-104580](https://github.com/godotengine/godot/pull/104580)).
+- certs: Sync with upstream as of Apr 8 2025 ([GH-106615](https://github.com/godotengine/godot/pull/106615)).
+- Fix unzSeekCurrentFile not resetting total_out_64 ([GH-106872](https://github.com/godotengine/godot/pull/106872)).
+- libwebm: Fix double free in mkvparser ContentEncoding ([GH-107781](https://github.com/godotengine/godot/pull/107781)).
+
+## Changelog
+
+**21 contributors** submitted around **45 fixes** for this release. See our [**interactive changelog**](https://godotengine.github.io/godot-interactive-changelog/#3.6.1) for the complete list of changes since the 3.6 release.
+
+This release is built from commit [`b1ba98fce`](https://github.com/godotengine/godot/commit/b1ba98fced19ac05b7a39b64a97dd7b1005cb7bb).
+
+<a id="downloads"></a>
+## Downloads
+
+The downloads for this dev snapshot can be found directly on our repository:
+
+- [Standard build](https://github.com/godotengine/godot-builds/releases/3.6.1) (GDScript, GDNative, VisualScript).
+- [Mono build](https://github.com/godotengine/godot-builds/releases/3.6.1) (C# support + all the above). You need to have dotnet CLI or MSBuild installed to use the Mono build. Relevant parts of Mono **6.12.0.182** are included in this build.
```

Comment:
Bad links, "page not found" for me. 🙃

Reply:
I opened this PR to correct the links:
https://github.com/godotengine/godot-website/pull/1106

It's kind of chicken and egg, I made the draft before I knew what the links would be.